magic6/magic6.cpp

```diff
@@ -0,0 +1,83 @@
+#include <fstream>
+
+using namespace std;
+
+ifstream input("magic6.in");
+ofstream output("magic6.out");
+
+char character;
+int n;
+int frequencyVector[10] = {};
+
+void verifyIfItsIntegerOrChar()
+{
+	if (character > 47 && character < 58)
+	{
+		frequencyVector[character - '0'] = 1;
+	}
+	else
+	{
+		character = (char)(character - 32);
+		output << character;
+	}
+}
+
+void writeNumbers()
+{
+	int poz = -1;
+	int i = 1;
+
+	if (frequencyVector[0] == 1)
+	{
+
+		while (poz < 0)
+		{
+			if (frequencyVector[i] == 1)
+			{
+				poz = i;
+			}
+			else
+			{
+				i++;
+			}
+		}
+		output << poz << 0;
+		for (int i = poz + 1; i < 10; i++)
+		{
+			if (frequencyVector[i] == 1)
+			{
+				output << i;
+			}
+		}
```

Comment:
duplicate code

Reply:
here i = poz + 1 but in the previous change request i = 1

Reply:
parameters!